github/pull_request.rb

```diff
@@ -105,6 +113,15 @@ def check_commits_style
     end
   end
 
+  def add_issue_links
+    if new? && issue_numbers.any?
+      message = issue_numbers.inject("\nIssues:") do |msg, issue_number|
+        msg + " [##{issue_number}](http://projects.theforeman.org/issues/#{issue_number})"
+      end
+      append_pull_request_body(message)
```

Comment:
I'm not sure we should be editing other users comments and posts directly. While it does create an additional notification, this would be altering a users comment and making it seem as if the user posted that? If I do a merge instead of squash or rebase does this edit get put into the commit message for the merge?